src/sqs.rs

```diff
@@ -185,24 +188,34 @@ mod test_send_messages_concurrently {
             .queue_url
             .unwrap();
 
-        // FIXME: use batch operations
         let mut results: Vec<usize> = vec![];
         while let Ok(x) = client
             .receive_message()
+            .max_number_of_messages(i32::try_from(MAX_MESSAGES).ok().unwrap())
             .wait_time_seconds(1)
             .queue_url(&queue_url)
             .send()
             .await
         {
             match x.messages {
                 Some(ref messages) => {
-                    assert_eq!(messages.len(), 1);
-                    let message = &messages[0];
-                    results.push(message.body.as_ref().unwrap().parse().unwrap());
+                    assert!(messages.len() <= MAX_MESSAGES);
+
+                    let results_delete = messages
+                        .iter()
+                        .map(|msg| {
+                            results.push(msg.body.as_ref().unwrap().parse().unwrap());
+                            DeleteMessageBatchRequestEntry::builder()
+                                .receipt_handle(msg.receipt_handle.as_ref().unwrap())
+                                .set_id(Some(msg.message_id.as_ref().unwrap().parse().unwrap()))
```

Comment:
I don't think the `parse` here does anything, IIUC `set_id` wants an `Option<String>` so you'll end up just parsing a string into a string.